### ISSUE TITLE
Document how to run from command-line

### ISSUE THREAD

**Issue by ericis** [spring-guides/gs-multi-module#3]
`./mvnw clean package spring-boot:run`

```
[ERROR] Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:1.5.2.RELEASE:run (default-cli) on project gs-multi-module-library: Unable t
o find a suitable main class, please add a 'mainClass' property -> [Help 1]
```

**Comment by gregturn** [comment]
That because it's multi module. At root can only build modules.

**Comment by dsyer** [comment]
I'm not sure why you would think that should work actually, since the application is defined in a subdirectory ("application"). You haven't installed the library in the local Maven repo either. I would expect you have to do `./mvnw install` first, and then `cd application; ../mvnw spring-boot:run`. Maybe we should include explicit instructions in the guide.

**Comment by dsyer** [comment]
This works for me:

```
$ ./mvnw install && ./mvnw spring-boot:run -pl application
```

**Comment by ericis** [comment]
I just wasn't thinking. Yeah, a little note might save people from themselves 👍

**Comment by dsyer** [comment]
Fixed in 6eca826
